README.md

```diff
@@ -1,104 +1,52 @@
 
-|                        | Open-source and Educational IDEs<sup>*</sup> | [CLion] | [IntelliJ IDEA] Ultimate, [GoLand] | [PyCharm] Professional | [WebStorm], [PhpStorm], other commercial IDEs |
-|------------------------|----------------------------------------------|---------|------------------------------------|------------------------|-----------------------------------------------|
-| [Language support]     | +                                            | +       | +                                  | +                      | +                                             |
-| [Cargo support]        | +                                            | +       | +                                  | +                      | +                                             |
-| [Code coverage]        | +                                            | +       | +                                  | +                      | +                                             |
-| [Detecting duplicates] | -                                            | +       | +                                  | +                      | +                                             |
-| [Debugger]             | -                                            | +       | +**                                | +**                    | -                                             |
-| [Run targets]          | -                                            | +       | +                                  | -                      | -                                             |
-| [Profiler]             | -                                            | +       | -                                  | -                      | -                                             |
-| [Valgrind Memcheck]    | -                                            | +       | -                                  | -                      | -                                             |
+As for now, plugin supports latest stable and the most previous stable version (although it can be changed in future).
+Consider using the original Rust plugin if you need to work with a version earlier than 2024.1; Consider contributing if this community plugin  does not enable support of the new IDEs fast enough.
 
+This plugin has been proven to work with:
+* PyCharm 2024.2 Community as well as Professional
 
-\* [IntelliJ IDEA] Community Edition, [PyCharm] Community Edition, [PyCharm Edu], [IntelliJ IDEA Edu], and third-party IntelliJ-based IDEs.
+### Advanced features
+There were several features present in the original plugin besides the original language support:
 
-\** Requires the
-[Native Debugging Support](https://plugins.jetbrains.com/plugin/12775-native-debugging-support) plugin. 
-LLDB only
+* debugging
+* profiler
+* valgrind
+* clion, appcode, etc extensions
 
-## TOML
+These features had been cut for now to make maintaining plugin up-to-date easier.
```

Comment:
Good point, but I was doing a little research lately and found some interesting stuff like lsp4j, so there is a possibility that someone from the community will implement its own version the api part that proprietary nativeDebugger exposes, so native debugger could be usable with community versions of the software (because keep in mind: debuggers are present in community ides, just not the native one, which I might guess is just some adapter between lldb/gdb <-> jdb, which is also exist as a separate community module (https://github.com/nativejdb/nativejdb + https://github.com/JetBrains/jdk-sa-jdwp) so it actually might be a question to erite couple connectors between different components).

As for the legal implications: I will check the jetbrains platform and what exactly exposes what and under which licenses and will return here with some info. Jetbrains just exposes some interfaces that code in this plugin implements so I'm gonna check license on the interfaces (but I assume they're clean)